include/zephyr/arch/riscv/csr.h

```diff
@@ -233,4 +233,39 @@
 				: "memory");			\
 })
 
+#define	uarthid		0x014	/* uhartid -- user hardware thread id, seems to be not yet defined in toolchain */
+
+#ifdef CONFIG_RISCV_KERNEL_IN_USER_MODE
+	/* zephyr runs in user mode */
+
+	/* inline assembly defines */
+	#define XSTATUS_IEN			USTATUS_IEN
+	#define XSTATUS_DEF_RESTORE		USTATUS_DEF_RESTORE
+	#define XSTATUS				0x000	// ustatus
```

Comment:
C++ style comments are generally not allowed in Zephyr

Reply:
understand, will fix this

Reply:
done